tests/sys_arduino_lib/main.cpp

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (C) 2019 Gunar Schorcht
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief    Demonstrates the use of an Arduino library imported as package
+ *
+ * @author   Gunar Schorcht <gunar@schorcht.net>
+ *
+ * @}
+ */
+
+#include <stdint.h>
+#include <TalkingLED.h>
+
+#include "arduino_board.h"
```

Comment:
```suggestion
#include "board.h"
```

And that should fix the Murdock failure.

Reply:
Thanks for the hint. I didn't look to Murdock's output after squashing since Murdock was happy before squashing. That is the problem arose after squashing.

I fixed and tested it again. I hope it is OK, that I pushed this minimal change already squashed.

Reply:
Hm, that did not the trick. Not all boards that provide the feature `arduino` also provide a definition of the `ARDUINO_LED`.

Reply:
I have added a default definition for `ARDUINO_LED`.
```
#ifndef ARDUINO_LED
#define ARDUINO_LED (13)    /* Arduino Uno LED pin */
#endif
```